sdk/go/core/tests/service_crud_test.go

```diff
@@ -3,7 +3,8 @@ package test
 import (
 	"fmt"
 	ysanity "github.com/CiscoDevNet/ydk-go/ydk/models/ydktest/sanity"
-	// "github.com/CiscoDevNet/ydk-go/ydk/models/ietf/interfaces"
```

Comment:
If this line isn't needed, please delete it.

Reply:
Done